publish/publish_workflow.wdl

```diff
@@ -50,6 +50,33 @@ task doMajorRelease {
         echo 'Assemble jars for cromwell and womtool'
         sbt -Dproject.version=~{releaseVersion} -Dproject.isSnapshot=false server/assembly womtool/assembly
 
+        echo 'Smoke test Cromwell and Womtool artifacts'
+        cat > hello.wdl <<FIN
+        task hello {
+          String name
+
+          command {
+            echo 'hello \${name}!'
```

Comment:
Um... does this work? Given the soup of escape sequences within WDLs within WDLs (and prior complaints about this being impossible), I wouldn't be surprised if this isn't doing what you expect it to...

Reply:
Yup it works. Getting this line right was the last piece of the puzzle. 😄